brainstormforce/astra
# AST-496 - fix: Added transition CSS to site-identity to make shrink sticky layout smoothen

### Description
- When the Shrink Effect option enables
- Headers & Site identity creates a jerk

### Screenshots
- https://share.bsf.io/yAur2qgw
- Fix - https://share.bsf.io/GGuWRjvp

### Types of changes
- Bugfix

### How has this been tested?
- Enable Sticky Header along with Shrink effect
- Notice the header height jerk after a slight scroll

### Checklist:
